### PR TITLE
PVR addon API 1.9.6 changes

### PR DESCRIPTION
**Pull request by ksooo**
https://github.com/kodi-pvr/pvr.hts/issues/13
https://github.com/kodi-pvr/pvr.hts/issues/14

Please test and report back.

Note: changelog.txt already contains description for the other PRs I want to include in 2.1.6 (#10, #11, #15)

### CODE REVIEW

File: src/HTSPTypes.h [hunk 1]
```diff
@@ -66,6 +67,7 @@ struct STag
   {
     del   = false;
     id    = 0;
+    index = 0;
```
Comment (Jalle19) [inline src/HTSPTypes.h]: You should probably merge #15 first and use the same convention here.

File: src/Tvheadend.cpp [hunk 2]
```diff
@@ -682,13 +683,24 @@ void CTvheadend::TransferEvent
...
+  epg.strWriter           = NULL; /* not supported by tvh */
+  epg.iYear               = 0;    /* not supported by tvh */
+  epg.strIMDBNumber       = NULL; /* not supported by tvh */
   epg.strIconPath         = event.image.c_str();
   epg.iGenreType          = event.content & 0xF0;
   epg.iGenreSubType       = event.content & 0x0F;
+  epg.strGenreDescription = NULL; /* not supported by tvh */
   epg.firstAired          = event.aired;
+  epg.iParentalRating     = 0; /* TODO: add support (seems to be only partially implemented => SEvent::age */
+  epg.iStarRating         = 0; /* TODO: add support (seems to be only partially implemented => SEvent::stars */
```
Comment (Jalle19) [inline src/Tvheadend.cpp]: These seem to be supported, although I don't have any EPG data with this information in order to test what values are actually returned by the API. See https://github.com/tvheadend/tvheadend/blob/master/src/htsp_server.c#L895

**Comment by Jalle19** [comment]
Thanks for the quick fixes! Could you perhaps add a message like `(closes #13` etc. to the commits, that way Github will automatically close the issues once merged?

File: src/HTSPTypes.h (hunk 1 above)
Comment (ksooo) [inline src/HTSPTypes.h]: Yes, that's plan.

File: src/Tvheadend.cpp (hunk 2 above)
Comment (ksooo) [inline src/Tvheadend.cpp]: Just wanted to leave a note in the code reminding me (or others) to fix it in another PR. Have that on my list now...

**Comment by ksooo** [comment]
Sure. Did not even know about this autoclose-feature.

**Comment by Jalle19** [comment]
Thanks, merge at will.

**Comment by ksooo** [comment]
@jalle19 merged. What needs to be updated in kodi repo to let it pick up the new pvr.hts version?

**Comment by xhaggi** [comment]
you need to adjust the commit SHA in
https://github.com/xbmc/xbmc/blob/master/project/cmake/addons/addons/pvr.hts/pvr.hts.txt

**Comment by xhaggi** [comment]
BTW @Jalle19 wow nice thanks a lot ;)

**Comment by xhaggi** [comment]
oops I have to say thank you @ksooo  .. thought the PR was from @Jalle19 ;)

**Comment by ksooo** [comment]
@xhaggi never mind.

**Pull request merged**